### PR TITLE
fix(remotecontrol): Replace info dialogs with notifications.

### PR DESCRIPTION
**Pull request by hristoterezov**

### CODE REVIEW

File: modules/UI/util/MessageHandler.js [hunk 1]
```diff
@@ -450,14 +450,14 @@ var messageHandler = {
      * @param messageArguments object with the arguments for the message.
      * @param options passed to toastr (e.g. timeOut)
      */
-    notify: function(displayName, displayNameKey, cls, messageKey,
-                     messageArguments, options) {
+    participantNotification: function(displayName, displayNameKey, cls,
```
Comment (virtuacoplenny) [inline modules/UI/util/MessageHandler.js]: Do you think it's worthwhile/straightforward to get this reactified now?

File: modules/UI/util/MessageHandler.js [hunk 2]
```diff
@@ -476,6 +476,35 @@ var messageHandler = {
...
+        const options = messageArguments
+            ? `data-i18n-options='${JSON.stringify(messageArguments)}'` : "";
+        let element = toastr.info(`
+            <span class="title" data-i18n="${titleKey}" />
+            <br>
+            <span data-i18n="${messageKey}" ${options} />
+            `
+        );
+        APP.translation.translateElement(element);
+        return element;
```
Comment (virtuacoplenny) [inline modules/UI/util/MessageHandler.js]: Why could you not try to reuse the existing notify? The two seem similar.

File: modules/UI/util/MessageHandler.js [hunk 3]
```diff
@@ -476,6 +476,35 @@ var messageHandler = {
...
+        // If we're in ringing state we skip all toaster notifications.
+        if(!notificationsEnabled || APP.UI.isOverlayVisible())
+            return;
+
+        const options = messageArguments
+            ? `data-i18n-options='${JSON.stringify(messageArguments)}'` : "";
+        let element = toastr.info(`
+            <span class="title" data-i18n="${titleKey}" />
+            <br>
+            <span data-i18n="${messageKey}" ${options} />
```
Comment (virtuacoplenny) [inline modules/UI/util/MessageHandler.js]: Are you certain these self closing tags are okay to use? void elements are common in jsx for elements that don't have content but there's only a handful for html.

File: modules/UI/util/MessageHandler.js (hunk 2 above)
Comment (hristoterezov) [inline modules/UI/util/MessageHandler.js]: Because it was more participant orientated.

**Pull request merged**